.gitignore

```diff
@@ -4,3 +4,4 @@
 bin
 .certs
 .tmp
+*.exe
```

Comment:
I'm not sure why we need to exclude win exe? Everything should be put under bin imo